tests/unit/lms/product/plugin/course_copy_test.py

```diff
@@ -0,0 +1,120 @@
+from unittest.mock import Mock, sentinel
+
+import pytest
+
+from lms.product.plugin.course_copy import CourseCopyFilesHelper
+from lms.services.exceptions import ExternalRequestError
+from tests import factories
+
+
+class TestCourseCopyFilesHelper:
+    @pytest.mark.parametrize(
+        "file,expected",
+        [
+            (Mock(course_id=sentinel.course_id), True),
+            (None, False),
+            (Mock(course_id=sentinel.other_course), False),
+        ],
+    )
+    def test_is_file_in_course(self, file_service, helper, file, expected):
+        file_service.get.return_value = file
+
+        is_in_course = helper.is_file_in_course(
+            file_service, sentinel.course_id, sentinel.file_id, sentinel.type
+        )
+
+        file_service.get.assert_called_once_with(sentinel.file_id, sentinel.type)
+        assert is_in_course == expected
+
+    @pytest.mark.parametrize("raising", [True, False])
+    def test_find_matching_file_in_course(self, helper, file_service, raising):
+        store_new_course_files = Mock()
```

Comment:
```suggestion
        store_new_course_files = create_autospec(lambda new_course_id: None)
```

Best to create a specced mock, otherwise the code could pass any number of arguments to `store_new_course_files()` and the test would still pass.